progressbar_example.py

```diff
@@ -0,0 +1,121 @@
+import porepy as pp
+import numpy as np
+from porepy.models.fluid_mass_balance import SinglePhaseFlow
+import logging
+from porepy.applications.md_grids.domains import nd_cube_domain
+
+logging.basicConfig(level=logging.INFO, format="%(message)s")
+
+
+class ModifiedGeometry:
```

Comment:
I think it is good practice to inherit from the base geometry. Do you agree, @IvarStefansson